packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -150,6 +150,9 @@ const mocks = (
           return harden({ chainId, stakingTokens });
         },
         async makeAccount() {
+          const { makeAccount: makeAcctErr } = errs;
+          if (makeAcctErr) throw makeAcctErr;
```

Comment:
The caller needs to be able to condition the error by chain name, much like in the transfer mock.

Also, they need to be able to turn it on and off. So perhaps it should be a map:

```suggestion
          const { makeAccount: makeAcctErr } = errs;
          if (makeAcctErr && makeAcctErr.get(name)) throw makeAcctErr.get(name);
```

Reply:
We need to simulate errors in `send` as well. And like `transfer`, the caller needs to be able to condition the failure on args. Sending 13 could be unlucky, perhaps.